specifyweb/frontend/js_src/lib/components/Atoms/Form.tsx

```diff
@@ -302,6 +302,7 @@ export const Textarea = wrap<
   `${className.notTouchedInput} ${className.textArea}`,
   ({ onValueChange, isReadOnly, ...props }) => ({
     ...props,
+    type: undefined,
```

Comment:
unset type attribute in case `props` accidentally contains it (happens if `{...getValidationAttributes(parser)}` was used in the code)

otherwise, typescript won't allow explicitly passing in `type` for textarea